### PR TITLE
Automated header cleanups

### PR DESCRIPTION
**Pull request by xeruf**
These are all purely automatic (Regex) changes that:
- remove copyright and filename headers (they were inconsistent anyways, and for file creation and authors we have a VCS)
- put `#pragma once` in place everywhere

I split them off from #3013, since that one unhelpfully threw them in with lots of other changes.

I took a lot of care to not delete any potentially useful information, that is why I preserved some comments that might not be ideally formatted or in the correct. However, these are manual edits that I want to avoid in code I don't know. When people touch the files again they will clearly see the information that was put there by the original creators (often a comment on the first line of the file) and can decide whether it can be deleted or where it should be incorporated.

Since this only concerns file headers (and sometimes footers), it won't mess up `git blame` either ;)
But it can save a lot of time from manually doing these things whenever such an old file is encountered.

### CODE REVIEW

**Comment by Holzhaus** [comment]
Can you rebase on 2.3? That should avoid merge conflicts later on.

**Comment by xerus2000** [comment]
I just rebased onto main :sweat_smile: I thought that is the branch that non-user-centric changes should go into?

**Comment by Holzhaus** [comment]
Yes, but this is basically a no-op and I worry about merge conflicts...

**Comment by xerus2000** [comment]
Done :+1: not quite a no-op, but nothing mention-worthy

**Comment by Holzhaus** [comment]
> not quite a no-op, but nothing mention-worthy

What do you mean? I thought this just removed some license/author comments and replace ifdefs with pragma once. What did you change? 

Please don't slip functional changes into "cleanup" PRs. Now I need to read through 5000 LOC to find out what was changed.

**Comment by daschuer** [comment]
> Please don't slip functional changes into "cleanup" PRs. Now I need to read through 5000 LOC to find out what was changed.

This is the reason we do not want mass refactoring for no reason. I think we have discussed this already in similar PRs from @xerus2000. There is only a minimum benefit is such PRs and it steals reviewer time that can be used for features and real bug fixes. 

Can we just close this?

**Comment by xerus2000** [comment]
> > not quite a no-op, but nothing mention-worthy
> 
> What do you mean? I thought this just removed some license/author comments and replace ifdefs with pragma once. What did you change?

I think we misunderstood no-op. I thought you referred to the act of rebasing being a no-op in the sense of there would be no conflicts, but they were a few due to pragma once-conversions in the meantime.

> Please don't slip functional changes into "cleanup" PRs. Now I need to read through 5000 LOC to find out what was changed.

No, I did not "slip in" anything. Nothing changed.

**Comment by xerus2000** [comment]
> There is only a minimum benefit is such PRs and it steals reviewer time that can be used for features and real bug fixes.

Integrating this with other changes steals away reviewer and coding time. I used some regexes to get this out of the way once and for all, because doing it by hand on the go became too tedious.

**Comment by Holzhaus** [comment]
> > > not quite a no-op, but nothing mention-worthy
> > 
> > What do you mean? I thought this just removed some license/author comments and replace ifdefs with pragma once. What did you change?
> 
> I think we misunderstood no-op. I thought you referred to the act of rebasing being a no-op in the sense of there would be no conflicts, but they were a few due to pragma once-conversions in the meantime.
> 
> > Please don't slip functional changes into "cleanup" PRs. Now I need to read through 5000 LOC to find out what was changed.
> 
> No, I did not "slip in" anything. Nothing changed.

OK, I misunderstood. The *PR* is still a functional no-op, but *rebase* was not because there were conflicts. Got it.

**Comment by xerus2000** [comment]
It would be good to merge this soon, since (trivial) conflicts arise quickly.

File: src/controllers/controller.h [hunk 1]
```diff
@@ -1,16 +1,11 @@
...
-* @file controller.h
-* @author Sean Pappalardo spappalardo@mixxx.org
-* @date Sat Apr 30 2011
-* @brief Base class representing a physical (or software) controller.
+/** Base class representing a physical (or software) controller.
 *
 * This is a base class representing a physical (or software) controller.  It
 * must be inherited by a class that implements it on some API. Note that the
 * subclass' destructor should call close() at a minimum.
 */
```
Comment (Holzhaus) [inline src/controllers/controller.h:6]: Use `///` and move in front of class.

File: src/controllers/controller.cpp [hunk 2]
```diff
@@ -1,9 +1,4 @@
-/**
-* @file controller.h
-* @author Sean Pappalardo spappalardo@mixxx.org
-* @date Sat Apr 30 2011
-* @brief Base class representing a physical (or software) controller.
-*/
+// Base class representing a physical (or software) controller.
```
Comment (Holzhaus) [inline src/controllers/controller.cpp:2]: Remove these two lines.

File: src/controllers/controllerenumerator.h [hunk 3]
```diff
@@ -1,15 +1,10 @@
...
-* @brief Base class handling discovery and enumeration of DJ controllers.
-*
-* This class handles discovery and enumeration of DJ controllers and
-*   must be inherited by a class that implements it on some API.
-*/
+/** Base class handling discovery and enumeration of DJ controllers.
+ *
+ * This class handles discovery and enumeration of DJ controllers and
+ *   must be inherited by a class that implements it on some API.
+ */
```
Comment (Holzhaus) [inline src/controllers/controllerenumerator.h:5]: Use `///` and move directly in front of class.

File: src/controllers/controllermanager.cpp [hunk 4]
```diff
@@ -1,9 +1,4 @@
-/**
-  * @file controllermanager.cpp
-  * @author Sean Pappalardo spappalardo@mixxx.org
-  * @date Sat Apr 30 2011
-  * @brief Manages creation/enumeration/deletion of hardware controllers.
-  */
+// Manages creation/enumeration/deletion of hardware controllers.
```
Comment (Holzhaus) [inline src/controllers/controllermanager.cpp:2]:
```suggestion
```

File: src/controllers/controllermanager.h [hunk 5]
```diff
@@ -1,12 +1,6 @@
-/**
-  * @file controllermanager.h
-  * @author Sean Pappalardo spappalardo@mixxx.org
-  * @date Sat Apr 30 2011
-  * @brief Manages creation/enumeration/deletion of hardware controllers.
-  */
+// Manages creation/enumeration/deletion of hardware controllers.
```
Comment (Holzhaus) [inline src/controllers/controllermanager.h:1]: Use `///` and move directly in front of class

File: src/controllers/controllerpresetfilehandler.cpp [hunk 6]
```diff
@@ -1,7 +1,4 @@
-/// @file controllerpresetfilehandler.cpp
-/// @author Sean Pappalardo spappalardo@mixxx.org
-/// @date Mon 9 Apr 2012
-/// @brief Handles loading and saving of Controller presets.
+/// Handles loading and saving of Controller presets.
```
Comment (Holzhaus) [inline src/controllers/controllerpresetfilehandler.cpp:2]:
```suggestion
```

File: src/controllers/controllerpresetfilehandler.h [hunk 7]
```diff
@@ -1,8 +1,5 @@
 #pragma once
-/// @file controllerpresetfilehandler.h
-/// @author Sean Pappalardo spappalardo@mixxx.org
-/// @date Mon 9 Apr 2012
-/// @brief Handles loading and saving of Controller presets.
+/// Handles loading and saving of Controller presets.
```
Comment (Holzhaus) [inline src/controllers/controllerpresetfilehandler.h:2]: move directly in front of class

File: src/controllers/controllerpresetinfo.h [hunk 8]
```diff
@@ -1,16 +1,11 @@
...
-* @file controllerpresetinfo.h
-* @author Ilkka Tuohela hile@iki.fi
-* @date Wed May 15 2012
-* @brief Base class handling enumeration and parsing of preset info headers
+/** Base class handling enumeration and parsing of preset info headers
 *
 * This class handles enumeration and parsing of controller XML description file
 * <info> header tags. It can be used to match controllers automatically or to
 * show details for a mapping.
 */
```
Comment (Holzhaus) [inline src/controllers/controllerpresetinfo.h:6]: Use `///` and move directly in front of class

File: src/controllers/controllerpresetinfo.cpp [hunk 9]
```diff
@@ -1,8 +1,4 @@
-/**
-* @file controllerpresetinfo.cpp
-* @author Ilkka Tuohela hile@iki.fi
-* @date Wed May 15 2012
-* @brief Implement handling enumeration and parsing of preset info headers
+/** Implement handling enumeration and parsing of preset info headers
```
Comment (Holzhaus) [inline src/controllers/controllerpresetinfo.cpp:1]: Redundant (see header), please remove.

File: src/controllers/controllerpresetinfoenumerator.cpp [hunk 10]
```diff
@@ -1,9 +1,4 @@
-/**
-* @file controllerpresetinfoenumerator.cpp
-* @author Be be.0@gmx.com
-* @date Sat Jul 18 2015
-* @brief Enumerate list of available controller mapping presets
-*/
+// Enumerate list of available controller mapping presets
```
Comment (Holzhaus) [inline src/controllers/controllerpresetinfoenumerator.cpp:1]:
```suggestion
```

File: src/controllers/controllerpresetinfoenumerator.h [hunk 11]
```diff
@@ -1,11 +1,5 @@
-/**
-* @file controllerpresetinfoenumerator.h
-* @author Be be.0@gmx.com
-* @date Sat Jul 18 2015
-* @brief Enumerate list of available controller mapping presets
-*/
-#ifndef CONTROLLERPRESETINFOENUMERATOR_H
-#define CONTROLLERPRESETINFOENUMERATOR_H
+// Enumerate list of available controller mapping presets
```
Comment (Holzhaus) [inline src/controllers/controllerpresetinfoenumerator.h:1]: Use `///` and move directly in front of class

File: src/controllers/dlgcontrollerlearning.cpp [hunk 12]
```diff
@@ -1,10 +1,4 @@
-/**
-* @file dlgcontrollerlearning.cpp
-* @author Sean M. Pappalardo  spappalardo@mixxx.org
-* @date Thu 12 Apr 2012
-* @brief The controller mapping learning wizard
-*
-*/
+// The controller mapping learning wizard
```
Comment (Holzhaus) [inline src/controllers/dlgcontrollerlearning.cpp:2]:
```suggestion
```

File: src/controllers/dlgcontrollerlearning.h [hunk 13]
```diff
@@ -1,12 +1,5 @@
-/**
-* @file dlgcontrollerlearning.h
-* @author Sean M. Pappalardo  spappalardo@mixxx.org
-* @date Thu 12 Apr 2012
-* @brief The controller mapping learning wizard
-*
-*/
-#ifndef DLGCONTROLLERLEARNING_H
-#define DLGCONTROLLERLEARNING_H
+// The controller mapping learning wizard
```
Comment (Holzhaus) [inline src/controllers/dlgcontrollerlearning.h:1]: Use `///` and move directly in front of class

File: src/controllers/dlgprefcontroller.cpp [hunk 14]
```diff
@@ -1,9 +1,4 @@
-/**
-* @file dlgprefcontroller.cpp
-* @author Sean M. Pappalardo  spappalardo@mixxx.org
-* @date Mon May 2 2011
-* @brief Configuration dialog for a DJ controller
-*/
+// Configuration dialog for a DJ controller
```
Comment (Holzhaus) [inline src/controllers/dlgprefcontroller.cpp:2]:
```suggestion
```

File: src/controllers/dlgprefcontroller.h [hunk 15]
```diff
@@ -1,10 +1,6 @@
-/// @file dlgprefcontroller.h
-/// @author Sean M. Pappalardo  spappalardo@mixxx.org
-/// @date Mon May 2 2011
-/// @brief Configuration dialog for a single DJ controller
+/// Configuration dialog for a single DJ controller
```
Comment (Holzhaus) [inline src/controllers/dlgprefcontroller.h:1]: move directly in front of class

File: src/controllers/hid/hidcontroller.cpp [hunk 16]
```diff
@@ -1,10 +1,4 @@
-/**
-  * @file hidcontroller.cpp
-  * @author Sean M. Pappalardo  spappalardo@mixxx.org
-  * @date Sun May 1 2011
-  * @brief HID controller backend
-  *
-  */
+// HID controller backend
```
Comment (Holzhaus) [inline src/controllers/hid/hidcontroller.cpp:2]:
```suggestion
```

File: src/controllers/hid/hidcontroller.h [hunk 17]
```diff
@@ -1,12 +1,6 @@
-/**
-  * @file hidcontroller.h
-  * @author Sean M. Pappalardo  spappalardo@mixxx.org
-  * @date Sun May 1 2011
-  * @brief HID controller backend
-  */
+// HID controller backend
```
Comment (Holzhaus) [inline src/controllers/hid/hidcontroller.h:1]: Use `///` and move directly in front of class

File: src/controllers/hid/hidcontrollerpreset.cpp [hunk 18]
```diff
@@ -1,7 +1,4 @@
-/// @file hidcontrollerpreset.cpp
-/// @author Jan Holthuis holzhaus@mixxx.org
-/// @date Mon 8 Apr 2020
-/// @brief HID/Bulk Controller Preset
+/// HID/Bulk Controller Preset
 ///
 /// This class represents a HID or Bulk controller preset, containing the data
 /// elements that make it up.
```
Comment (Holzhaus) [inline src/controllers/hid/hidcontrollerpreset.cpp:4]: move into header file and put it directly in front of class

File: src/controllers/midi/hss1394controller.h [hunk 19]
```diff
@@ -1,8 +1,4 @@
-/**
-  * @file hss1394controller.h
-  * @author Sean M. Pappalardo  spappalardo@mixxx.org
-  * @date Thu 15 Mar 2012
-  * @brief HSS1394-based MIDI backend
+/** HSS1394-based MIDI backend
```
Comment (Holzhaus) [inline src/controllers/midi/hss1394controller.h:1]: Use `///` and move directly in front of class

File: src/controllers/midi/hss1394enumerator.h [hunk 20]
```diff
@@ -1,12 +1,6 @@
-/**
-* @file hss1394enumerator.h
-* @author Sean Pappalardo spappalardo@mixxx.org
-* @date Thu 15 Mar 2012
-* @brief This class handles discovery and enumeration of DJ controllers that appear under the HSS1394 cross-platform API.
-*/
+// This class handles discovery and enumeration of DJ controllers that appear under the HSS1394 cross-platform API.
```
Comment (Holzhaus) [inline src/controllers/midi/hss1394enumerator.h:1]: Use `///` and move directly in front of class

File: src/controllers/midi/midicontroller.cpp [hunk 21]
```diff
@@ -1,10 +1,4 @@
-/**
- * @file midicontroller.cpp
- * @author Sean Pappalardo spappalardo@mixxx.org
- * @date Tue 7 Feb 2012
- * @brief MIDI Controller base class
- *
- */
+// MIDI Controller base class
```
Comment (Holzhaus) [inline src/controllers/midi/midicontroller.cpp:2]:
```suggestion
```

File: src/controllers/midi/midicontroller.h [hunk 22]
```diff
@@ -1,17 +1,12 @@
...
-* @author Sean Pappalardo spappalardo@mixxx.org
-* @date Tue 7 Feb 2012
-* @brief MIDI Controller base class
+/** MIDI Controller base class
 *
 * This is a base class representing a MIDI controller.
 *   It must be inherited by a class that implements it on some API.
 *
 *   Note that the subclass' destructor should call close() at a minimum.
 */
```
Comment (Holzhaus) [inline src/controllers/midi/midicontroller.h:7]: Use `///` and move directly in front of class

File: src/controllers/midi/midicontrollerpreset.cpp [hunk 23]
```diff
@@ -1,7 +1,4 @@
-/// @file midicontrollerpreset.cpp
-/// @author Jan Holthuis holzhaus@mixxx.org
-/// @date Wed 8 Apr 2020
-/// @brief MIDI Controller Preset
+/// MIDI Controller Preset
```
Comment (Holzhaus) [inline src/controllers/midi/midicontrollerpreset.cpp:1]: Move this whole comment into the header file (directly in front of class)

File: src/controllers/midi/midicontrollerpresetfilehandler.cpp [hunk 24]
```diff
@@ -1,10 +1,4 @@
-/**
-* @file midicontrollerpresetfilehandler.cpp
-* @author Sean Pappalardo spappalardo@mixxx.org
-* @date Mon 9 Apr 2012
-* @brief Handles loading and saving of MIDI controller presets.
-*
-*/
+// Handles loading and saving of MIDI controller presets.
```
Comment (Holzhaus) [inline src/controllers/midi/midicontrollerpresetfilehandler.cpp:2]:
```suggestion
```

File: src/controllers/midi/midicontrollerpresetfilehandler.h [hunk 25]
```diff
@@ -1,12 +1,6 @@
-/**
- * @file midicontrollerpresetfilehandler.h
- * @author Sean Pappalardo spappalardo@mixxx.org
- * @date Mon 9 Apr 2012
- * @brief Handles loading and saving of MIDI controller presets.
- */
+// Handles loading and saving of MIDI controller presets.
```
Comment (Holzhaus) [inline src/controllers/midi/midicontrollerpresetfilehandler.h:1]: Use `///` and move directly in front of class

File: src/controllers/midi/midioutputhandler.h [hunk 26]
```diff
@@ -1,15 +1,10 @@
-/**
- * @file midioutputhandler.h
- * @author Sean Pappalardo spappalardo@mixxx.org
- * @date Tue 11 Feb 2012
- * @brief Static MIDI output mapping handler
+/** Static MIDI output mapping handler
  *
  * This class listens to a control object and sends a midi message based on the
  * value.
  */
```
Comment (Holzhaus) [inline src/controllers/midi/midioutputhandler.h:5]: Use `///` and move directly in front of class

File: src/controllers/midi/portmidicontroller.cpp [hunk 27]
```diff
@@ -1,11 +1,4 @@
-/**
- * @file portmidicontroller.h
- * @author Albert Santoni alberts@mixxx.org
- * @author Sean M. Pappalardo  spappalardo@mixxx.org
- * @date Thu 15 Mar 2012
- * @brief PortMidi-based MIDI backend
- *
- */
+// PortMidi-based MIDI backend
```
Comment (Holzhaus) [inline src/controllers/midi/portmidicontroller.cpp:2]:
```suggestion
```

File: src/controllers/midi/portmidicontroller.h [hunk 28]
```diff
@@ -1,9 +1,4 @@
-/**
- * @file portmidicontroller.h
- * @author Albert Santoni alberts@mixxx.org
- * @author Sean M. Pappalardo  spappalardo@mixxx.org
- * @date Thu 15 Mar 2012
- * @brief PortMidi-based MIDI backend
+/** PortMidi-based MIDI backend
```
Comment (Holzhaus) [inline src/controllers/midi/portmidicontroller.h:1]: Use `///` and move directly in front of class

File: src/encoder/encoder.h [hunk 29]
```diff
@@ -1,14 +1,6 @@
-/****************************************************************************
-                   encoder.h  - encoder API for mixxx
-                             -------------------
-    copyright            : (C) 2009 by Phillip Whelan
-    copyright            : (C) 2010 by Tobias Rafreider
-    copyright            : (C) 2017 by Josep Maria Antolín
- ***************************************************************************/
+// encoder API for mixxx
```
Comment (Holzhaus) [inline src/encoder/encoder.h:1]: This comment doesn't add any value, please remove

File: src/encoder/encoderbroadcastsettings.cpp [hunk 30]
```diff
@@ -1,9 +1,4 @@
-/**
-* @file encoderbroadcastsettings.cpp
-* @author Josep Maria Antolín
-* @date Feb 27 2017
-* @brief storage of broadcast settings for the encoders.
-*/
+// storage of broadcast settings for the encoders.
```
Comment (Holzhaus) [inline src/encoder/encoderbroadcastsettings.cpp:2]:
```suggestion
```

File: src/encoder/encoderbroadcastsettings.h [hunk 31]
```diff
@@ -1,13 +1,6 @@
-/**
-* @file encoderbroadcastsettings.h
-* @author Josep Maria Antolín
-* @date Feb 27 2017
-* @brief storage of broadcast settings for the encoders.
-*/
+// storage of broadcast settings for the encoders.
```
Comment (Holzhaus) [inline src/encoder/encoderbroadcastsettings.h:1]: Use `///` and move directly in front of class

File: src/encoder/encoderffmpegcore.h [hunk 32]
```diff
@@ -1,15 +1,5 @@
...
-    copyright            : (C) 2012-2013 by Tuukka Pasanen
-                           (C) 2007 by Wesley Stessens
-                           (C) 1994 by Xiph.org (encoder example)
-                           (C) ???? Tobias Rafreider (broadcast and recording fixes)
- ***************************************************************************/
-
-
-#ifndef ENCODERFFMPEGCORE_H
-#define ENCODERFFMPEGCORE_H
+// core code for ffmpeg encoders
```
Comment (Holzhaus) [inline src/encoder/encoderffmpegcore.h:1]: Use `///` and move directly in front of class

File: src/encoder/encoderffmpegmp3.cpp [hunk 33]
```diff
@@ -1,21 +1,4 @@
...
-/***************************************************************************
- *                                                                         *
- *   This program is free software; you can redistribute it and/or modify  *
- *   it under the terms of the GNU General Public License as published by  *
- *   the Free Software Foundation; either version 2 of the License, or     *
- *   (at your option) any later version.                                   *
- *                                                                         *
- ***************************************************************************/
-
+// FFMPEG MP3 encoder for mixxx
```
Comment (Holzhaus) [inline src/encoder/encoderffmpegmp3.cpp:1]:
```suggestion
```

File: src/encoder/encoderffmpegmp3.h [hunk 34]
```diff
@@ -1,23 +1,5 @@
...
- *   This program is free software; you can redistribute it and/or modify  *
- *   it under the terms of the GNU General Public License as published by  *
- *   the Free Software Foundation; either version 2 of the License, or     *
- *   (at your option) any later version.                                   *
- *                                                                         *
- ***************************************************************************/
-
-#ifndef ENCODERFFMPEGMP3_H
-#define ENCODERFFMPEGMP3_H
+// FFMPEG MP3 encoder for mixxx
```
Comment (Holzhaus) [inline src/encoder/encoderffmpegmp3.h:1]: Use `///` and move directly in front of class (and remove "for Mixxx")

File: src/encoder/encoderffmpegvorbis.cpp [hunk 35]
```diff
@@ -1,21 +1,4 @@
...
-/***************************************************************************
- *                                                                         *
- *   This program is free software; you can redistribute it and/or modify  *
- *   it under the terms of the GNU General Public License as published by  *
- *   the Free Software Foundation; either version 2 of the License, or     *
- *   (at your option) any later version.                                   *
- *                                                                         *
- ***************************************************************************/
-
+// FFMPEG OGG/Vorbis encoder for mixxx
```
Comment (Holzhaus) [inline src/encoder/encoderffmpegvorbis.cpp:1]:
```suggestion
```

File: src/encoder/encoderffmpegvorbis.h [hunk 36]
```diff
@@ -1,23 +1,5 @@
...
- *   This program is free software; you can redistribute it and/or modify  *
- *   it under the terms of the GNU General Public License as published by  *
- *   the Free Software Foundation; either version 2 of the License, or     *
- *   (at your option) any later version.                                   *
- *                                                                         *
- ***************************************************************************/
-
-#ifndef ENCODERFFMPEGVORBIS_H
-#define ENCODERFFMPEGVORBIS_H
+// FFMPEG OGG/Vorbis encoder for mixxx
```
Comment (Holzhaus) [inline src/encoder/encoderffmpegvorbis.h:1]: Use `///` and move directly in front of class (and remove "for mixxx")

File: src/encoder/encoderflacsettings.cpp [hunk 37]
```diff
@@ -1,10 +1,4 @@
-/**
-* @file encoderflacsettings.cpp
-* @author Josep Maria Antolín
-* @date Feb 27 2017
-* @brief storage of setting for flac encoder
-*/
-
+// storage of setting for flac encoder
```
Comment (Holzhaus) [inline src/encoder/encoderflacsettings.cpp:2]:
```suggestion
```

File: src/encoder/encoderflacsettings.h [hunk 38]
```diff
@@ -1,12 +1,6 @@
-/**
-* @file encoderflacsettings.cpp
-* @author Josep Maria Antolín
-* @date Feb 27 2017
-* @brief storage of setting for flac encoder
-*/
+// storage of setting for flac encoder
```
Comment (Holzhaus) [inline src/encoder/encoderflacsettings.h:1]: Use `///` and move directly in front of class

File: src/encoder/encodermp3settings.cpp [hunk 39]
```diff
@@ -1,9 +1,4 @@
-/**
-* @file encodermp3settings.cpp
-* @author Josep Maria Antolín
-* @date Feb 27 2017
-* @brief storage of setting for mp3 encoder
-*/
+// storage of setting for mp3 encoder
```
Comment (Holzhaus) [inline src/encoder/encodermp3settings.cpp:2]:
```suggestion
```

File: src/encoder/encodermp3settings.h [hunk 40]
```diff
@@ -1,12 +1,6 @@
-/**
-* @file encodermp3settings.cpp
-* @author Josep Maria Antolín
-* @date Feb 27 2017
-* @brief storage of setting for mp3 encoder
-*/
+// storage of setting for mp3 encoder
```
Comment (Holzhaus) [inline src/encoder/encodermp3settings.h:1]: Use `///` and move directly in front of class

File: src/encoder/encodersettings.h [hunk 41]
```diff
@@ -1,12 +1,6 @@
-/**
-* @file encodersettings.cpp
-* @author Josep Maria Antolín
-* @date Feb 27 2017
-* @brief Encoder settings interface for encoders.
-*/
+// Encoder settings interface for encoders.
```
Comment (Holzhaus) [inline src/encoder/encodersettings.h:1]: Use `///` and move directly in front of class

File: src/encoder/encodersndfileflac.cpp [hunk 42]
```diff
@@ -1,9 +1,4 @@
-/**
-* @file encodersndfileflac.cpp
-* @author Josep Maria Antolín
-* @date Feb 27 2017
-* @brief encoder for flac using libsndfile
-*/
+// encoder for flac using libsndfile
```
Comment (Holzhaus) [inline src/encoder/encodersndfileflac.cpp:2]:
```suggestion
```

File: src/encoder/encodersndfileflac.h [hunk 43]
```diff
@@ -1,14 +1,6 @@
...
-* @file encodersndfileflac.h
-* @author Josep Maria Antolín
-* @date Feb 27 2017
-* @brief encoder for flac using libsndfile
-*/
-
-
-#ifndef ENCODERSNDFILEFLAC_H
-#define ENCODERSNDFILEFLAC_H
+// encoder for flac using libsndfile
```
Comment (Holzhaus) [inline src/encoder/encodersndfileflac.h:1]: Use `///` and move directly in front of class

File: src/encoder/encodervorbis.cpp [hunk 44]
```diff
@@ -1,30 +1,19 @@
...
-*/
+// vorbis encoder for mixxx
+// It's a real mess right now.
+//
+// When I get around to cleaning things up,
+// I'll probably make an Encoder base class,
+// so we can easily add in an EncoderLame (or something) too.
+//
+// A lot of stuff has been stolen from:
+// http://svn.xiph.org/trunk/vorbis/examples/encoder_example.c
```
Comment (Holzhaus) [inline src/encoder/encodervorbis.cpp:9]: Remove or move to header file

File: src/encoder/encodervorbis.h [hunk 45]
```diff
@@ -1,14 +1,5 @@
...
-                             -------------------
-    copyright            : (C) 2007 by Wesley Stessens
-                           (C) 1994 by Xiph.org (encoder example)
-                           (C) 1994 Tobias Rafreider (broadcast and recording fixes)
- ***************************************************************************/
-
-
-#ifndef ENCODERVORBIS_H
-#define ENCODERVORBIS_H
+// vorbis encoder for mixxx
```
Comment (Holzhaus) [inline src/encoder/encodervorbis.h:1]: Use `///` and move directly in front of class (and remove "for mixxx")

File: src/encoder/encodervorbissettings.cpp [hunk 46]
```diff
@@ -1,9 +1,4 @@
-/**
-* @file encoderwavesettings.cpp
-* @author Josep Maria Antolín
-* @date Feb 27 2017
-* @brief storage of setting for vorbis encoder
-*/
+// storage of setting for vorbis encoder
```
Comment (Holzhaus) [inline src/encoder/encodervorbissettings.cpp:2]:
```suggestion
```

File: src/encoder/encodervorbissettings.h [hunk 47]
```diff
@@ -1,12 +1,6 @@
-/**
-* @file encoderwavesettings.h
-* @author Josep Maria Antolín
-* @date Feb 27 2017
-* @brief storage of setting for vorbis encoder
-*/
+// storage of setting for vorbis encoder
```
Comment (Holzhaus) [inline src/encoder/encodervorbissettings.h:1]: Use `///` and move directly in front of class

File: src/encoder/encoderwave.cpp [hunk 48]
```diff
@@ -1,9 +1,4 @@
-/**
-* @file encoderwave.cpp
-* @author Josep Maria Antolín
-* @date Feb 27 2017
-* @brief wave/aiff "encoder" for mixxx
-*/
+// wave/aiff "encoder" for mixxx
```
Comment (Holzhaus) [inline src/encoder/encoderwave.cpp:2]:
```suggestion
```

File: src/encoder/encoderwave.h [hunk 49]
```diff
@@ -1,13 +1,6 @@
-/**
-* @file encoderwave.h
-* @author Josep Maria Antolín
-* @date Feb 27 2017
-* @brief wave/aiff "encoder" for mixxx
-*/
-
-#ifndef ENCODERWAVE_H
-#define ENCODERWAVE_H
+// wave/aiff "encoder" for mixxx
```
Comment (Holzhaus) [inline src/encoder/encoderwave.h:1]: Use `///` and move directly in front of class

File: src/encoder/encoderwavesettings.cpp [hunk 50]
```diff
@@ -1,12 +1,8 @@
-/**
-* @file encoderwavesettings.cpp
-* @author Josep Maria Antolín
-* @date Feb 27 2017
-* @brief storage of setting for wave/aiff encoder
-*/
+// storage of setting for wave/aiff encoder
```
Comment (Holzhaus) [inline src/encoder/encoderwavesettings.cpp:2]:
```suggestion
```

File: src/encoder/encoderwavesettings.h [hunk 51]
```diff
@@ -1,12 +1,6 @@
-/**
-* @file encoderwavesettings.h
-* @author Josep Maria Antolín
-* @date Feb 27 2017
-* @brief storage of setting for wave/aiff encoder
-*/
+// storage of setting for wave/aiff encoder
```
Comment (Holzhaus) [inline src/encoder/encoderwavesettings.h:1]: Use `///` and move directly in front of class

File: src/engine/channels/engineaux.cpp [hunk 52]
```diff
@@ -1,5 +1,3 @@
-// engineaux.cpp
-// created 4/8/2011 by Bill Good (bkgood@gmail.com)
 // shameless stolen from enginemicrophone.cpp (from RJ)
```
Comment (Holzhaus) [inline src/engine/channels/engineaux.cpp:2]:
```suggestion
```

File: src/engine/channels/engineaux.h [hunk 53]
```diff
@@ -1,9 +1,6 @@
-// engineaux.h
-// created 4/8/2011 by Bill Good (bkgood@gmail.com)
 // unapologetically copied from enginemicrophone.h from RJ
```
Comment (Holzhaus) [inline src/engine/channels/engineaux.h:2]:
```suggestion
```

**Review by Holzhaus — CHANGES_REQUESTED** [review]

**Comment by xerus2000** [comment]
> I took a lot of care to not delete any potentially useful information, that is why I preserved some comments that might not be ideally formatted or in the correct. However, these are manual edits that I want to avoid in code I don't know. When people touch the files again they will clearly see the information that was put there by the original creators (often a comment on the first line of the file) and can decide whether it can be deleted or where it should be incorporated.

@Holzhaus I don't want to do any manual edits here

**Comment by Holzhaus** [comment]
> > I took a lot of care to not delete any potentially useful information, that is why I preserved some comments that might not be ideally formatted or in the correct. However, these are manual edits that I want to avoid in code I don't know. When people touch the files again they will clearly see the information that was put there by the original creators (often a comment on the first line of the file) and can decide whether it can be deleted or where it should be incorporated.
> 
> @Holzhaus I don't want to do any manual edits here

I don't really see the purpose then tbh.

> But it can save a lot of time from manually doing these things whenever such an old file is encountered.

Deleting 10 lines is 3 keystrokes for me (`d9j`). Cleaning up the comments takes a bit longer and would actually benefit during development because doxygen comments are picked up by editors/IDEs.

**Comment by Holzhaus** [comment]
Closing in favor of https://github.com/mixxxdj/mixxx/pull/3397.

**Pull request closed without merging**